### PR TITLE
Admin drivers view

### PR DESCRIPTION
**Pull request by michaelFakhry**

### CODE REVIEW

File: app/adapters/admin.js [hunk 1]
```diff
@@ -48,4 +48,89 @@ export default Ember.Service.extend({
       url: this.env.APP.apiUrl + this.namespace + '/rides?pageSize=' + this.env.APP.adminRidesPageSize + '&page=' + page + attributes
     });
   },
+  /**
```
Comment (michaelFakhry) [inline app/adapters/admin.js]: drivers endpoints

File: app/pods/components/admin-driver-detail/component.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
+import Ember from 'ember';
```
Comment (michaelFakhry) [inline app/pods/components/admin-driver-detail/component.js]: driver detail component used in admin and driver web apps

File: app/pods/components/driver-table/component.js [hunk 3]
```diff
@@ -45,7 +45,7 @@ export default Ember.Component.extend({
```
Comment (michaelFakhry) [inline app/pods/components/driver-table/component.js]: component that holds a driver table .. used each time we need to view a table of drivers

File: app/pods/components/editable-driver-credentials-form/component.js [hunk 4]
```diff
@@ -0,0 +1,194 @@
+import Ember from 'ember';
```
Comment (michaelFakhry) [inline app/pods/components/editable-driver-credentials-form/component.js]: editable driver credentials form used in admin and driver web apps

File: app/pods/components/editable-driver-profile-form/component.js [hunk 5]
```diff
@@ -0,0 +1,50 @@
+import Ember from 'ember';
```
Comment (michaelFakhry) [inline app/pods/components/editable-driver-profile-form/component.js]: driver profile form used in both admin and driver web apps

File: app/pods/components/editable-driver-vehicle-information-form/component.js [hunk 6]
```diff
@@ -0,0 +1,157 @@
+import Ember from 'ember';
```
Comment (michaelFakhry) [inline app/pods/components/editable-driver-vehicle-information-form/component.js]: driver vehicle information form used in both admin and driver web apps

File: app/pods/components/expandable-ride-table/component.js [hunk 7]
```diff
@@ -0,0 +1,15 @@
+import Ember from 'ember';
```
Comment (michaelFakhry) [inline app/pods/components/expandable-ride-table/component.js]: a component that holds the a ride table .. but this table is expandable with 'view more' rather name numbered pagination

**Comment by msalahz** [comment]
@michaelFakhry Can you please follow this points amd make sure they are applied in your code review :

File: app/pods/components/editable-driver-vehicle-information-form/template.hbs [hunk 8]
```diff
@@ -0,0 +1,97 @@
+<div id="step4" class="wizard_content" style="display: block;">
```
Comment (engpeter2010) [inline app/pods/components/editable-driver-vehicle-information-form/template.hbs]: The component name is too long. Why not simply rename it to "vehicle-information-form"?

File: app/pods/components/expandable-ride-table/component.js (hunk 7 above)
Comment (engpeter2010) [inline app/pods/components/expandable-ride-table/component.js]: Nice! View more is more user friendly option than pagination.

**Pull request merged**